lib/http/service.js

```diff
@@ -18,6 +18,11 @@ module.exports = (container) => {
   ////////////////////////////////////////////////////////////////////////////////
   // PRERESOURCE MIDDLEWARE
 
+  service.use((req, res, next) => {
+    req.headers['x-request-received-at'] = Date.now();
```

Comment:
Is storing this as a request header idiomatic?  I'd expect request headers to have come from the client.